pages/settings/SettingsLayout.qml

```diff
@@ -304,10 +305,6 @@ Rectangle {
             i += 1;
         }
 
-        fiatPriceProviderDropDown.update();
```

Comment:
<img width="342" alt="Screenshot 2020-05-01 at 14 37 28" src="https://user-images.githubusercontent.com/7697454/80805815-49b48780-8bb9-11ea-8181-ae5675b91b0f.png">

Reply:
Can't reproduce. Do you see any errors in the logs? Could you try clean build?
Does the dropdown contain any options? What happens if you select one and restart the GUI?

Reply:
> Do you see any errors in the logs?

No

>  Could you try clean build?

Same issue

> Does the dropdown contain any options?

Yes, the dropdown looks correct

> What happens if you select one and restart the GUI?

Same issue.

Reply:
Note that the other dropdown menus are correct.

Reply:
Fixed. Thanks for catching this.